src/software/firmware/srcs/pressure_controller.cpp

```diff
@@ -141,6 +144,11 @@ void PressureController::endRespiratoryCycle() {
             onPeakPressureIncrease(min(plateauDiff, MAX_PEAK_INCREMENT));
         }
     }
+
+    // RCM-SW-18
+    if (m_pressure <= ALARM_THRESHOLD_MAX_PRESSURE) {
```

Comment:
Should this test be against `m_pressure`? 
This is run only one per cycle, so m_pressure value will be the latest measure during respiratory cycle.

Reply:
This only for the "undetection". Detection happens in the `compute()` method.

Reply:
My concern is about a peak above `ALARM_THRESHOLD_MAX_PRESSURE` and a drop under `ALARM_THRESHOLD_MAX_PRESSURE`during PEEP. Should we triggered the alarm?

Reply:
I think so, yes. If the point is to avoid hurting lungs because the pressure is too high, alarm should ring at least ~1 cycle, right?

Reply:
I'm ok with your solution.